docs/userguide.md

```diff
@@ -7,8 +7,18 @@ is a walkthrough tutorial for how to deploy the federation v2 control plane.
 
 ### Binaries
 
-The federation deployment depends on `apiserver-boot`, `crinit`, and `kubectl`
-being installed in the path. These binaries can be installed via the
+Federation can be deployed with a prebuilt image from federation-v2 master using `scripts/deploy-federation-latest.sh`.
+The federation deployment depends on [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) being installed in the path.
```

Comment:
How about give end user a recommended version of kubectl? I think we are currently using 1.11 in `download-binaries.sh`, but we should able to support kubectl versions which support CRD. In order not to bring trouble, how about specify 1.9 as minimum version for kubectl?